whylog/tests/tests_client/__init__.py

```diff
@@ -35,13 +35,91 @@ def test_one(self, test_name):
         output_path = path_join(path, 'expected_output.txt')
 
         whylog_base = WhylogBase(parser=parsers_path, rules=rules_path)
-        whylog_client = WhylogClient(base=whylog_base, open_path=path)
+        # whylog_client = WhylogClient(rulesbase=whylog_base, open_path=path)
 
         with open(input_path, 'r') as f:
             vim_line = f.read()
-        result = whylog_client.get_cause(vim_line)
+        # TODO call get_cause with sens...
+        # result = whylog_client.get_cause(1, vim_line)
 
         with open(output_path, 'r') as f:
             pass
             # remove this comment after implement whylog client and base
             # assert result == f.read()
+
+class TestBacktrackSearcher(TestCase):
+
+    def test_imports_correctness(self):
+        FOO_BAR = "/foo/bar"
+        client = WhylogClient(rulesbase=WhylogBase())
+        bts = searchers.BacktrackSearcher(file_path=FOO_BAR)
+        assert bts._file_path == FOO_BAR
```

Comment:
this is not a part of the interface, why do you test it? Such test is not very useful but will break when refactoring.